test/client/fixtures/sandbox/code-instrumentation/getters-test.js

```diff
@@ -525,3 +525,63 @@ test('we should clean up html and remove extra namespaces from svg (GH-1083)', f
 
     strictEqual(getProperty(div, 'innerHTML'), nativeDiv.innerHTML);
 });
+
+test('we should not process element\'s properties if they do not exist (GH-1164)', function () {
+    var div             = document.createElement('div');
+    var svg             = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+    var html            = '<a href="/path">link</a>';
+    var processedHtml   = processHtml(html);
+    var style           = 'body {}';
+    var processedStyle  = styleProcessor.process(style, null, true);
+    var textProperties  = ['innerText', 'text', 'textContent'];
+
+    div.appendChild(svg);
+
+    var svgHasInnerHTML = svg.innerHTML !== void 0;
```

Comment:
Lets' change test logic as following:
 - there are native elements and processed elements
- if native element contains text or html property then overridden element should returns processed value
- else if it should return origin value